Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm

```diff
@@ -239,6 +249,81 @@ void requestLinkDecorationFilteringData(LinkFilteringRulesCallback&& callback)
     }];
 }
 
+StorageAccessPromptQuirkController& StorageAccessPromptQuirkController::shared()
+{
+    static StorageAccessPromptQuirkController* sharedInstance = new StorageAccessPromptQuirkController;
```

Comment:
`MainThreadNeverDestroyed` (from Chris' review comment in the [other PR](https://github.com/WebKit/WebKit/pull/21292/files)).